lib/optimizely/helpers/network_utils.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+#
+#    Copyright 2020, Optimizely and contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'net/http'
+
+module Optimizely
+  module Helpers
+    module NetworkUtils
```

Comment:
@thomaszurkan-optimizely Please suggest a name. 
`NetworkUtils` is quite broader term. `HttpUtils` would be more appropriate here.